packages/SwingSet/package.json

```diff
@@ -15,7 +15,7 @@
   },
   "scripts": {
     "build": "exit 0",
-    "test": "ava",
+    "test": "ava --node-arguments=--expose-gc",
```

Comment:
Let's put this in the ava section instead, as in #3136 's https://github.com/Agoric/agoric-sdk/pull/3136/files#diff-0a8e846b29355fbc43138cf62c2ed73752a727b81450d3c3ae857a04de128d66 . If I manage to land #3136 today, one of our copies will disappear in a rebase, whereas adding it to the test script would effectively set the argument twice.

Reply:
I had no idea that was even a thing.  Cool.  I will make that change.